test/fixtures/test-runner/output/retryable.js

```diff
@@ -0,0 +1,60 @@
+const common = require('../../../common');
+const test = require('node:test');
+const assert = require('node:assert');
+
```

Comment:
I think we need a test case where a test (not suite) has retries set and there are several subtests that also have retries, but the last subtest never passes. We need to make sure there is no duplicated reporting.